bindings/protocols/coap/coap.ttl

```diff
@@ -0,0 +1,90 @@
+@prefix cov: <https://www.w3.org/2019/wot/coap#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@base <https://www.w3.org/2019/wot/coap> .
+
+<> a owl:Ontology ;
+   dcterms:title "CoAP Vocabulary for the Web of Things"@en ;
+   rdfs:label "CoAP Vocabulary for the Web of Things"@en ; # TODO: Refactor
+   # TODO: Also add dctrms:creator?
+   dcterms:author [ a dcterms:Agent ,
```

Comment:
There is no `dcterms:author` in [DCMI Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/).